examples/cdi-basic/README_pt.adoc

```diff
@@ -0,0 +1,169 @@
+= CDI @Inject
+:index-group: CDI
+:jbake-type: page
+:jbake-status: published
+
+Para usar `@Inject`, a primeira coisa que você precisa de um arquivo `META-INF/beans.xml` arquivo no modulo
```

Comment:
word "arquivo" two times

Reply:
Daniel,
I have to use double quotes "",  in did of using simple ''?

small change :   
   -  change  "Add README.pt.adoc Translate to portuguese: examples cdi basic TOMEE " to "Translate to portuguese: examples cdi basic TOMEE-2516"
    - insert small description with link of JIRA

I did not understand! 

Would you help me, please.

Reply:
word "arquivo" two times  :  

change : 

Para usar `@Inject`, a primeira coisa que você precisa de um arquivo `META-INF/beans.xml` arquivo no modulo

to 

Para usar `@Inject`, a primeira coisa que você precisa de um arquivo `META-INF/beans.xml`  no modulo

Reply:
change "Add README.pt.adoc Translate to portuguese: examples cdi basic TOMEE " to "Translate to portuguese: examples cdi basic TOMEE-2516"

change title of your PR to  => Translate to portuguese: examples cdi basic TOMEE-2516

and add the link of your ticket JIRA

Reply:
Ok ,
I will strive to improve next time.
Thank you very much

Reply:
great, thanks my friend :  )